crates/re_viewer/src/misc/caches/mod.rs

```diff
@@ -18,18 +20,22 @@ pub struct Caches {
 
 impl Caches {
     /// Call once per frame to potentially flush the cache(s).
-    pub fn new_frame(&mut self) {
+    pub fn begin_frame(&mut self) {
         let max_image_cache_use = 1_000_000_000;
-        self.image.new_frame(max_image_cache_use);
+        let max_decode_cache_use = 1_000_000_000;
```

Comment:
Since the decode cache is RAM-only, we could make it quite a bit bigger. Maybe 8 GB?

```suggestion
        let max_decode_cache_use = 8_000_000_000;
```

Reply:
but Web!! If we go there, this needs to have a different limit on Web

Reply:
good catch - yeah, 8GB is quite a high limit for a 4GB system :)

Reply:
8 still seems high in general. Splitting the difference and going with 4 normally and keeping it as 1 for wasm.